src/skins/telefonica.tsx

```diff
@@ -181,7 +182,7 @@ export const getTelefonicaSkin: GetKnownSkin = () => {
             appBarBackground: palette.darkModeGrey,
             background: palette.darkModeBlack,
             backgroundContainer: palette.darkModeGrey,
-            backgroundBrand: palette.darkModeBlack,
```

Comment:
Should be backgroundBrand kept instead of being replaced?

Reply:
yes!